contributing/code_of_conduct/care_team.rst

```diff
@@ -1,30 +1,30 @@
-Enforcement Team
-================
+CARE Team
+=========
 
 Our Pledge
 ----------
 
-In the interest of fostering an open and welcoming environment, the enforcement team
-pledge to ensure that the spirit of the :doc:`Code of Conduct </contributing/code_of_conduct/index>`
+In the interest of fostering an open and welcoming environment, the CoC Active Response Ensurers, or CARE,
+pledged to ensure that the spirit of the :doc:`Code of Conduct </contributing/code_of_conduct/index>`
 is respected. Our main priority is to ensure the safety of our community members.
 The second goal is to help educate the community as a whole to be aware of the CoC
 and how to help implement its spirit throughout the community. In case these goals
 conflict, we will prioritize safety of community members over all other goals.
 
 If you think there is or has been a violation to the code of conduct please contact
-enforcement team or if you prefer contact only individual members of the enforcement team.
+the CARE team or if you prefer contact only individual members of the CARE team.
 
 Members
 -------
 
-Here are all the members of the Code of Conduct enforcement team. You can contact
+Here are all the members of the Code of Conduct CARE team. You can contact
 any of them directly using the contact details below or you can also contact all of
 them at once by emailing **coc@symfony.com**.
 
-About the Enforcement Team
+About the CARE Team
 --------------------------
```

Comment:
The RST syntax is a bit picky ... so we need to make this `------` line as long as the new heading. Thanks!